source/develop/release-management/features/metrics/metrics-store-installation.html.md

```diff
@@ -122,63 +84,101 @@ This procedure will define the curator pod so that it deletes metrics indexes th
 
        ovirt-metrics-<ovirt_env_name>:
         delete:
-         days: 7
-          runhour: 0
-           runminute: 0
+          days: 3
 
 
-5. Update "ovirt_env_name". **Use the same name you configured earlier**.
+5. Update <ovirt_env_name>. **Use the same name you configured earlier**.
 6. Run
 
         # oc rollout latest dc/logging-curator
         # oc rollout status -w dc/logging-curator
 
 
-
 ## oVirt Hypervisors and Engine Setup ##
 
-oVirt machines on versions 4.1.1 and above include fluentd and collectd packages.
 Now we need to deploy and configure collectd and fluentd to send the data to the central Metrics Store:
 
-1. Install / Upgrade and setup oVirt Engine 4.1.3 and above.
+1. Requires installing / upgrading and setup oVirt Engine 4.1.8 and above.
+   Until 4.1.8 is available, You can use the snapshots repository to get the latest ovirt-engine-metrics package.
+   http://resources.ovirt.org/pub/ovirt-4.1-snapshot/rpm/
 
-2. Install / Upgrade and activate one or more hosts, 4.1.3 and above.
+2. Requires installing / upgrading and activating one or more hosts, 4.1.8 and above.
+   Until 4.1.8 is available, You can use the snapshots repository to get the latest ovirt-engine-metrics package.
+   http://resources.ovirt.org/pub/ovirt-4.1-snapshot/rpm/
 
-3. Copy the CA certificate - created earlier on in this procedure [(see oVirt Metrics Store Setup)](https://github.com/ViaQ/Main/blob/master/README-install.md#getting-the-shared_key-and-ca-cert) - to the engine machine.
+3. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
 
+   On the oVirt engine machine, run:
 
-   On the metrics store machine, run:
+        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine-metrics/config.yml
 
-        # scp /path/to/ca/certificate.cert root@<fqdn/of/engine/machine>:/etc/ovirt-engine-metrics
+4. Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:
 
-4. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
+        # vi /etc/ovirt-engine-metrics/config.yml
 
-   On the oVirt engine machine, run:
+5. Update the values with your specific environment details:
+
+
+- `ovirt_env_name:` **Use the same name you configured earlier**.
+
+  Environment name. Can be used to identify data collected in a single central
```

Comment:
"Environment name. It can be used to identify data collected in a single central"

Reply:
Done